android/KMEA/app/src/main/assets/keyboard.html

```diff
@@ -98,8 +98,24 @@
           break;
       }
 
-      kbdInterface.registerStub({KN:keyboardName,KI:'Keyboard_'+internalName,KLC:langId,KL:languageName,
-        KF:kbdFile,KFont:font,KOskFont:oskFont,KP:package});
+      var k = {
+        KN:keyboardName,
+        KI:'Keyboard_'+internalName,
+        KLC:langId,
+        KL:languageName,
+        KF:kbdFile,
+        KP:package
+      };
+      if (font) {
+        k.KFont = font;
+        window.console.log('keyboard.html assigning k.KFont = ' + font);
```

Comment:
Let's take the logging out of the final build

Reply:
right. I've added a trello card for that